README.mkdn

```diff
@@ -59,8 +59,8 @@ The repo is laid out as follows.
 
 # Developing
 
-We currently support Linux and Windows as first-tier platforms. macOS is also
-used on a daily basis by Oxide employees, but is not tested in CI.  The build
+We currently support Linux as first-tier platforms. macOS is also
+used on a daily basis by Oxide and Rivos employees.  The build
 probably also works on Illumos; if anyone would like to step up to maintain
 support and a continuous build for Illumos or macOS, we'd love the help.
```

Comment:
Nit: the reference to macOS could be removed here.

Reply:
IDK, we still need people to test stuff on mac as the CI builds are not necessarily hermetic.